### PR TITLE
push 1.1.6 patch

### PR DESCRIPTION
**Pull request by erohmensing**
**Proposed changes**:
- release 1.1.6 which fixes the docs tutorials 

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -6,24 +6,38 @@ Rasa Change Log
...
-  has been updated from ``False`` or ``None`` to an empty list (``[]``)
 
 
 Removed
 -------
 
 
+Fixed
+-----
+
```
Comment (wochinge) [inline CHANGELOG.rst]: should I remove the empty ones?

Comment (erohmensing) [reply CHANGELOG.rst]: I removed the empties from 1.1.6, I thought we keep them for the new master version

Comment (wochinge) [reply CHANGELOG.rst]: omg, its late

Comment (erohmensing) [reply CHANGELOG.rst]: Don't stay to late 😘

**Pull request merged**